### PR TITLE
adding perlin noise functionality

### PR DESCRIPTION
**Pull request by roz3x**

### CODE REVIEW

File: noise/noise.go [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 package noise
 
 import (
 	"image"
+	"image/color"
 	"math/rand"
 	"time"
 
 	"github.com/anthonynsimon/bild/parallel"
+	"github.com/roz3x/bild/perlin"
```
[the remark this answers is not in the document]
Comment (roz3x) [reply noise/noise.go]: okay ! would be done soon .

File: main.go [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
 
-import "github.com/anthonynsimon/bild/cmd"
+import (
+	"image/png"
+	"os"
+
+	"github.com/roz3x/bild/noise"
+)
 
 func main() {
```
[the remark this answers is not in the document]
Comment (roz3x) [reply main.go]: sorry my bad,  i changed it for testing purpose .

[the remark this answers is not in the document]
Comment (roz3x) [reply main.go]: okay i will delete the output folder ,  but  where would i host the images then ?!

**Comment by anthonynsimon** [comment]
I will host them with the others, no worries :)

**Comment by roz3x** [comment]
thanks ! but you have to change the link in README too .

**Comment by anthonynsimon** [comment]
@roz3x looks great 🎉 thanks a lot for your help!

**Comment by roz3x** [comment]
hey Thanks !

**Pull request merged**